teencode-org/teencode
# allow for easy transition of blog posts

*What does this PR Do?*
* Allows the blog article page show loading icon and noticeable transition when a suggested article is clicked.
* improves on the existing 'suggested article' frontend logic

*How to test*
* Visit the `/blog` route, click on an article
* Scroll down, Confirm that the suggested articles are unique to themselves and the current page article
* Click on one of the suggested articles
* Confirm the page loading icon shows up before the newer page shows up.
